### PR TITLE
fix: ACNA-1375 - updater for runtimes.json for aio app run --local

### LINKED ISSUE #491: updater for `runtimes.json` for `aio app run --local`
For this file: https://github.com/adobe/aio-cli-plugin-app/blob/master/bin/openwhisk-standalone-config/runtimes.json

Docker registry calls:
1. https://registry.hub.docker.com/v2/repositories/adobeapiplatform/adobe-action-nodejs-v14/tags
2. https://registry.hub.docker.com/v2/repositories/adobeapiplatform/adobe-action-nodejs-v12/tags
3. https://registry.hub.docker.com/v2/repositories/adobeapiplatform/adobe-action-nodejs-v10/tags

Each call returns a json object. The `results[0].name` property from that json object will have the latest tag for each.

### PR DESCRIPTION
**Pull request by shazron**
## Description

Closes #491

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: bin/openwhisk-standalone-config/get-runtimes.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    const output = {
+      runtimes: {
+        nodejs
+      }
+    }
+    console.log(JSON.stringify(output, null, 2))
+  }
+
+
+  main()
```
Comment (moritzraho) [inline bin/openwhisk-standalone-config/get-runtimes.js:52]: new line :)

**Review by moritzraho — COMMENTED** [review]
lgtm, but trying to understand when get-runtimes gets called

File: bin/openwhisk-standalone-config/get-runtimes.js (hunk 1 above)
Comment (shazron) [reply bin/openwhisk-standalone-config/get-runtimes.js]: fixed :)

**Comment by shazron** [comment]
> lgtm, but could you give me more insights on when get runtimes is called

I added a npm script that one of the team can run to update the json. This is a manual process, I'm not sure if it should be run  automatic for the user (and cached) for local development, although it might be a good idea

**Comment by moritzraho** [comment]
Ok do this automatically would be better, but maybe as a future step, we could open an issue ?
wdyt ?

**Comment by shazron** [comment]
> Ok do this automatically would be better, but maybe as a future step, we could open an issue ? wdyt ?

yeah let's do that for the future

**Comment by shazron** [comment]
@moritzraho  filed the issue for the automatic update https://github.com/adobe/aio-cli-plugin-app/issues/558

**Pull request merged**